Source/Core/InputCommon/ControllerEmu/Setting/NumericSetting.h

```diff
@@ -69,9 +83,11 @@ class NumericSettingBase
   const char* GetUIName() const;
   const char* GetUISuffix() const;
   const char* GetUIDescription() const;
+  const bool IsAdvanced() const;
```

Comment:
```suggestion
  bool IsAdvanced() const;
```

Since top-level const on a value type being returned doesn't really do much

Reply:
👍